x/axelarnet/keeper/keeper.go

```diff
@@ -151,23 +178,39 @@ func (k Keeper) GetCosmosChains(ctx sdk.Context) []string {
 	defer utils.CloseLogError(iter, k.Logger(ctx))
 
 	for ; iter.Valid(); iter.Next() {
-		bz := iter.Value()
-		results = append(results, string(bz))
+		var value types.CosmosChain
+		iter.UnmarshalValue(&value)
+		results = append(results, value.Name)
 	}
 
 	return results
 }
 
+func (k Keeper) getCosmosChain(ctx sdk.Context, chain string) (types.CosmosChain, bool) {
+	key := cosmosChainPrefix.Append(utils.LowerCaseKey(chain))
+	var value types.CosmosChain
+	ok := k.getStore(ctx).Get(key, &value)
+	return value, ok
```

Comment:
```suggestion
func (k Keeper) getCosmosChain(ctx sdk.Context, chain string) (cosmosChain types.CosmosChain, ok bool) {
        return cosmosChain, k.getStore(ctx).Get(cosmosChainPrefix.Append(utils.LowerCaseKey(chain)), &cosmosChain)
```
Just some one-liner trick I've been doing. Don't have to do it if you don't like it. 😛